genesyscloud/routing_queue/resource_genesyscloud_routing_queue_members_test.go

```diff
@@ -0,0 +1,174 @@
+package routing_queue
+
+import (
+	"context"
+	"fmt"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"testing"
+
+	"github.com/mypurecloud/platform-client-sdk-go/v154/platformclientv2"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUpdateQueueUserRingNum(t *testing.T) {
```

Comment:
Do you mind refactoring this to `TestUnitUpdateQueueUserRingNum` ? That was the -run regex filter will pick it up as a unit test

Reply:
Done @charliecon